modules/docs/arrow-docs/docs/docs/arrow/core/try/README.md

```diff
@@ -93,8 +93,8 @@ lotteryTry.getOrElse { emptyList() }
 If you want to perform a check on a possible success, you can use `filter` to convert successful computations in failures if conditions aren't met:
 
 ```kotlin:ank
```

Comment:
You can turn this into a `kotlin:ank:playground`
And add costume `message` to your Exceptions to get your `PredicateException(message=Predicate does not hold for $exceptionVariable)`